### PR TITLE
refresh app detail page pt.2, app info (bug 1103201)

### PR DESCRIPTION
**Pull request by ngokevin**
Depends on https://github.com/mozilla/fireplace/pull/873

- Still working on polish and kinks.
- Need to refresh and refactor receipt page since that uses detail styles

screen shot 2014-12-31 at 6 38 05 am

screen shot 2014-12-31 at 6 38 14 am

screen shot 2014-12-31 at 6 38 30 am

### CODE REVIEW

**Comment by ngokevin** [comment]
@spasovski

File: src/templates/detail/main.html [hunk 1]
```diff
@@ -45,83 +62,105 @@
...
-{# No placeholder, the block below already supplies one #}
-{% except %}
-{% end %}
+  <section class="main full app-info">
+    <div class="detail-flex-wrap">
+      <div class="detail-left-side">
+        {% if this.banner_regions and this.banner_message and
+              this.banner_regions.indexOf(user_helpers.region()) != -1 %}
+          <section class="region-banner">
+            <div>HELLO</div>
```
Comment (spasovski) [inline src/templates/detail/main.html]: ???

**Comment by spasovski** [comment]
Looks good. Mobile issue I saw earlier seems fixed too.

File: src/templates/detail/main.html (hunk 1 above)
Comment (ngokevin) [inline src/templates/detail/main.html]: oops

**Pull request closed without merging**